TWLight/users/helpers/editor_data.py

```diff
@@ -4,14 +4,27 @@
 import typing
 import urllib.request, urllib.error, urllib.parse
 from django.conf import settings
-from django.utils.timezone import now
+from django.utils import timezone
 
 logger = logging.getLogger(__name__)
 
 
 def editor_global_userinfo(
     wp_username: str, wp_sub: typing.Optional[int], strict: bool
 ):
+    """
+    Public interface for fetching Editor global_userinfo.
+    Parameters
+    ----------
+    wp_username : str
```

Comment:
Since you are already adding a docstring, can you include a brief description of each parameter?

Reply:
Done!